src/codegeneration/TemplateLiteralTransformer.js

```diff
@@ -283,6 +267,9 @@ export class TemplateLiteralTransformer extends TempVarTransformer {
           binaryExpression = binaryExpression.right;
       }
       let transformedTree = this.transformAny(tree.elements[i]);
+      if (element.type === TEMPLATE_SUBSTITUTION) {
+        transformedTree = parseExpression `String(${transformedTree})`;
```

Comment:
String() is not really correct (won't throw on symbols) but I dunno if it matters much